dockers/docker-base/no_install_recommend_suggest

```diff
@@ -0,0 +1,5 @@
+# Instruct apt-get to NOT install "recommended" or "suggested" packages by
+# default when installing a package.
+
+APT::Install-Recommends "false";
```

Comment:
Consider merge with files/apt/apt.conf.d/81norecommends, and share the same file in both host and docker images.

Reply:
Good idea.